site/content/contribute/more-info/webapp/e2e/which-query-to-use.md

```diff
@@ -1,63 +0,0 @@
----
-title: "Which query to use?"
```

Comment:
It appears this content is being deleted without being added to any other locations. Is it necessary? Based on [this conversation](https://community.mattermost.com/core/pl/r71a83zdufdcprs66cuxh9k9dy) it appears the content is missing a substantial volume of information, so I think it might be worth considering removal until we have something to replace it, but I'm 0/5

Reply:
Hey @BenLloydPearson :) I believe I condensed the content from this page into the file `site/content/contribute/more-info/webapp/e2e-cheatsheets.md`